plugins/equalizer/__init__.py

```diff
@@ -27,6 +27,7 @@
 
 import logging
 import os
+import math
```

Comment:
Sorting alphabetically :-)

```suggestion
import math
import os
```

Reply:
Nice catch, thanks!